### PR TITLE
Fixed some typos and formulations

### PR DESCRIPTION
**Pull request by Faberen**

### CODE REVIEW

**Review by annahj1006 — APPROVED** [review]
Looks great. Can see i wrote "bellow" many times xd

File: docs/Report.md [hunk 1]
```diff
@@ -41,14 +41,15 @@ At the outermost layer, we end with our SQL-Server and razor pages, which intera
...
 
 You will see in our repositories, that we're deleting the author at some point, this was a project demand. We had two possibilities; delete the user in the sense that they will no longer be traceable, that is make everything anonymous and delete their information, or we had the possibility of deleting everything that the user ever created. 
-We chose to be sure that the user wouldn't come back complaining that their username/normal name still was in a cheep, so we deleted everything that they created.  We chose to give the user full control and ownership over their content, so we deleted everything that they had created. 
+We chose to be sure that the user wouldn't come back complaining that their username/normal name still was in a cheep, so we deleted everything that they created. By doing this the user has full control over the deletion of their data. 
+
```
Comment (AlbinoLoaf) [inline docs/Report.md:48]: Good rewrite

File: docs/Report.md [hunk 2]
```diff
@@ -41,14 +41,15 @@ At the outermost layer, we end with our SQL-Server and razor pages, which intera
...
+The method IncreaseLikeAttribute in the CheepRepository, which can be seen in the diagram [@fig:InfrastructureUML], reveals that like is an attribute on the Cheep entity since its only parameters are a Cheep id and not an author id. This is the simplest implementation of the feature, we could come up with. We have chosen to use this implementation due to the overall time constraint of the project. It has the impact, that it is not possible to see or retrieve data from the database, about who has liked a cheep. It is possible for each author to like multiple times. It is not possible to regret a like in the current state of the application, although a dislike method could be implemented.
```
Comment (AlbinoLoaf) [inline docs/Report.md:52]: Nice catch

File: docs/Report.md [hunk 3]
```diff
@@ -59,9 +60,9 @@ The method IncreaseLikeAttribute in the CheepRepository, which can be seen in th
 
 ![UML Class Pages](Images/PackagePagesUMLDiagram.png){width=60% #fig:PagesUML}
 
-The Onion Architecture (otherwise known as Clean Architecture), is great for having low coupling and high cohesion. When looking at the UML in the more specified onion diagram bellow, there is no unnecessary communication between scripts. 
+The Onion Architecture (otherwise known as Clean Architecture), is great for having low coupling and high cohesion. When looking at the UML in the more specified onion diagram below, there is no unnecessary communication between scripts.
```
Comment (AlbinoLoaf) [inline docs/Report.md:63]: Dont reference position, reference the figure

File: docs/Report.md [hunk 4]
```diff
@@ -92,16 +91,15 @@ If a new user wants register they will need to login with github. Since this wil
...
+The database sends the 32 cheeps to the repository. Which inserts each cheep into a CheepDTO before returning a list of 32 CheepDTOs. This list is sent back through the system, shown in [@fig:SQD1]. Arriving in Chirp.Razor. It is weaved into the HTML, checking if the user is Authorized. Before the page is returned to the UA. 
 
 ![Sequence Diagram Unauthorized](Images/SequenceDiagramUnauthorized.png){width=70% #fig:SQD1}
 
-[@fig:SQD2] show a known actor accessing our site, logging in and sending a Cheep. The first Get request is the same as seen in [@fig:SQD1]. It deviates during the authentication step as the actor presses the login link. As the actor logs log in, Microsoft Identity redirects them to Azure OIDC. Which then redirect to GitHub. 
+[@fig:SQD2] show a known actor accessing our site, logging in and sending a Cheep. The first Get request is the same as seen in [@fig:SQD1]. It deviates during the authentication step as the actor presses the login link. As the actor logs in, Microsoft Identity redirects using the OIDC protocl with the B2C tenant login flow. Which then redirect to GitHub.
```
Comment (AlbinoLoaf) [inline docs/Report.md:102]: Good and clear alteration

**Review by AlbinoLoaf — APPROVED** [review]
Overall your  corrections look good

**Review by SkarpKat — APPROVED** [review]

**Review by Lukan707 — APPROVED** [review]
Very nice

**Pull request merged**